src/Media_Command.php

```diff
@@ -174,6 +174,9 @@ function regenerate( $args, $assoc_args = array() ) {
 	 * [--desc=<description>]
 	 * : "Description" field (post content) of attachment post.
 	 *
+	 * [--import_only]
```

Comment:
What do you think about calling this `--skip-copy` or similar, to better communicate that we're skipping existing behavior?